src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -130,24 +129,42 @@ export interface IScatterPlotSampleData {
     profiledMutations?:boolean;
     mutations: AnnotatedMutation[];
     copyNumberAlterations: AnnotatedNumericGeneMolecularData[];
+    xThresholdType?: '>'|'<'|undefined; // threshold symbol for x coordinate in scatter plot
+    yThresholdType?: '>'|'<'|undefined; // threshold symbol for y coordinate in scatter plot
+    thresholdType?: '>'|'<'|undefined; // threshold symbol for continuous axis in box-scatter and waterfall plot
```

Comment:
please put this separately into IBoxScatterPlotPoint and IWaterfallPlotSampleData

Reply:
See discussion in the comment above.

Reply:
Resoved in rework of interfaces.